users.go

```diff
@@ -816,7 +816,8 @@ func (s *UsersService) GetEmail(email int, options ...RequestOptionFunc) (*Email
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/projects.html#add-email
 type AddEmailOptions struct {
-	Email *string `url:"email,omitempty" json:"email,omitempty"`
+	Email            *string `url:"email,omitempty" json:"email,omitempty"`
+	SkipConfirmation *bool   `url:"skiconfirmation,omitempty" json:"skiconfirmation,omitempty"`
```

Comment:
```suggestion
	SkipConfirmation *bool   `url:"skip_confirmation,omitempty" json:"skip_confirmation,omitempty"`
```

Reply:
I'm really sorry. I don't known what i was thinking at when i've committed it !
But it is very weird. It works !
Anyway, I'll fix it in a minute.